core/client/app/mixins/current-user-settings.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 var CurrentUserSettings = Ember.Mixin.create({
     currentUser: function () {
```

Comment:
what if this method died? as long as you're reducing the number of alias's for session.user ...